lib/Twig/Extension/Core.php

```diff
@@ -1257,7 +1257,7 @@ function twig_test_empty($value)
  *
  * <pre>
  * {# evaluates to true if the foo variable is an array or a traversable object #}
```

Comment:
Should this be updated as well then?

Reply:
Is it the only occurrence that needs to be changed in the doc?

Reply:
Well, we should update the short description of the function 3 lines above too (what it checks is for being iterable, not traversable, according to PHP's naming)

Reply:
The doc itself uses the right name